src/EventApi.php

```diff
@@ -243,6 +243,17 @@ public function persist(CalendArtAbstractEvent $event)
             $method = 'post';
         }
 
+        // manage the options now
+        // should we send notifications to the attendees ?
+        if (isset($options['sendNotifications'])) {
+            $url = sprintf(
+                '%s?sendNotifications=%s',
+                $url,
+                $options['sendNotifications'] ? 'true' : 'false'
+            );
```

Comment:
not a fan of building the url like this. You should use the guzzle's query object